webui/src/pages/repositories/repository/pulls/pullDetails.jsx

```diff
@@ -32,18 +31,12 @@ const PullDetailsContent = ({repo, pull}) => {
         setError(null);
         setLoading(true);
         try {
-            await refsAPI.merge(repo.id, pull.source_branch, pull.destination_branch);
+            await pullsAPI.merge(repo.id, pull.id);
         } catch (error) {
             setError(`Failed to merge pull request: ${error.message}`);
```

Comment:
I think the error here may be confusing. The merge might have been successful and the failure was on the update.
Since we are returning a verbose error message from the server, lets just use it as is.

Reply:
Right! It's there since we had two different errors (merge + status change), and now it isn't relevant anymore.
Changed.